source/SIL.AppBuilder.Portal.Frontend/src/ui/routes/project/__tests__/acceptance-test.tsx

```diff
@@ -0,0 +1,126 @@
+import { describe, it, beforeEach } from '@bigtest/mocha';
+import { visit, location } from '@bigtest/react';
+import { expect } from 'chai';
+
+import { setupApplicationTest, setupRequestInterceptor, useFakeAuthentication } from 'tests/helpers/index';
+
+
+import page from './page';
+
+describe.only('Acceptance | Archive Project', () => {
+  setupApplicationTest();
+  setupRequestInterceptor();
+  useFakeAuthentication();
+
+  beforeEach(function () {
+
+    this.mockGet(200, 'organizations', {
+      data: [{
+        type: 'organizations',
+        id: 1,
+        attributes: {}
+      }]
+    });
+
+    this.mockGet(200, 'projects/1', {
+      data: {
+        type: 'projects',
+        id: '1',
+        attributes: {
+          'date-archived': null,
+          'date-created': "2018-08-09T20:23:54.809962",
```

Comment:
not all attributes need to be specified :)

Reply:
though, if we had a factory library... maybe that'd help out. 
been thinking about switching all the polly stuff to `@bigtest/mirage`